Tests/StoreKitUnitTests/StoreTransactionTests.swift

```diff
@@ -71,7 +71,7 @@ class StoreTransactionTests: StoreKitConfigTestCase {
 
         let sk2Transaction = try await self.createTransactionWithPurchase()
 
-        let transaction = StoreTransaction(sk2Transaction: sk2Transaction)
+        let transaction = StoreTransaction(sk2Transaction: sk2Transaction, jwsRepresentation: "")
```

Comment:
Maybe pass an actual JWS (UUID for example) and check `transaction.jwsRepresentation`?
`testSK1DetailsWrapCorrectly` also needs a `expect(transaction.jwsRepresentation).to(beNil())`.